Sprint-2/extend/format-time.js

```diff
@@ -2,13 +2,37 @@
 // Your task is to write tests for as many different groups of input data or edge cases as you can, and fix any bugs you find.
 
 function formatAs12HourClock(time) {
-  const hours = Number(time.slice(0, 2));
+  let hours = Number(time.slice(0, 2));
+  let minutes = Number(time.slice(3, 5));
+  //"Cant use NUmber function on minutes because if minutes is 00 it defaults to a single 0 which we cannot use in time to display the minutes" So I need to create an if else statement to take care of these cased myself.
+  if ((minutes < 10 && minutes >= 0)) {
+    minutes = `0${minutes}`;
+  }
+  else if (minutes >= 10 && minutes <= 59) {
+    minutes = minutes;
+  }
+  else {
+    "The minutes on the time is incorrect and out of the normal range."
+  }
+
   if (hours > 12) {
-    return `${hours - 12}:00 pm`;
+    return `${hours - 12}:${minutes} pm`;
+  }
+  else if (hours == Number("00")) {
```

Comment:
Yes that line is necessary in this case so that I don't have to create an hourStr variable. It checks for the numeric value because hours is a variable that it being passed number values

Reply:
Why not just write `else if (hours == 0) {` ?